Gemfile

```diff
@@ -38,7 +38,7 @@ source 'https://rubygems.org' do
   gem 'postgres-copy'
   gem 'puma', '~> 3.0'
   gem 'pusher'
-  gem 'rails', '~> 5.1.1'
+  gem 'rails', '~> 5.1.2'
```

Comment:
This isn't strictly necessary. Can we leave it as-is and let `bundle update rails` do its thing?